std/os/nspire.zig

```diff
@@ -0,0 +1,20 @@
+pub const STDIN_FILENO = 0;
+pub const STDOUT_FILENO = 1;
+pub const STDERR_FILENO = 2;
+
+pub use @import("linux/errno.zig");
+const c = @import("../c.zig");
+
+pub fn getErrno(r: usize) usize {
+    const signed_r = @bitCast(isize, r);
+    return if (signed_r > -4096 and signed_r < 0) @intCast(usize, -signed_r) else 0;
+}
+
+pub fn write(fd: i32, buf: [*]const u8, count: usize) usize {
+    return @intCast(usize, c.write(fd, buf, count));
+}
+
+pub fn read(fd: i32, buf: *c_void, count: usize) usize {
+    return @intCast(usize, c.read(fd, buf, count));
```

Comment:
Have you read the c sources and seen what it would take to do this without them?

Reply:
It looks like it's a syscall (see [syscall.h](https://github.com/ndless-nspire/Ndless/blob/master/ndless-sdk/include/syscall.h) and [syscall-list.h](https://github.com/ndless-nspire/Ndless/blob/master/ndless-sdk/include/syscall-list.h)). We could probably use that asm to call the syscalls straight from zig, without needing the libc!